subsys/Kconfig

```
@@ -56,3 +56,7 @@ rsource "ieee802154/Kconfig"
 rsource "dm/Kconfig"
 
 rsource "nrf_security/Kconfig"
+
+rsource "audio/a_mod/lc3_t2/decoder/Kconfig"
+
+rsource "a_mod/Kconfig"
```

```suggestion
rsource "audio_module/Kconfig"
```
This is a global comment; I don't think Amod is very descriptive. I suggest using audio_module